docs/reference/contributing/guidelines/workflow.md

```diff
@@ -52,7 +52,7 @@ Pull requests on GitHub have to meet the following requirements to keep the code
 - All new features and enhancements require documentation, tests and user guides for us to accept them. Please link each pull request to all relevant documentation and test pull requests.
 - Avoid merging commmits. (Always rebase when possible.)
 - Comment in the pull request on every change (rebase or new commits). This helps reviewers to be up to date with changes
-- Pull requests should fix a bug, add a feature or refactor.
+- Pull requests should fix a bug, add a feature or refactor. The only exception are third-party party version updates (for instance MbedTLS or Nanostack releases for Mbed OS).
```

Comment:
party there twice

Reply:
Maybe also add:
These should considered as a library update and contain an updated version, good level of detail documenting what has changed and any impacts on the user.

Reply:
updated.